ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -107,15 +139,14 @@ function mapStateToProps (state) {
   return {
     sortBy: state.appVersionsSortDirection,
     columns: state.appVersionsColumns,
-    appVersions: state.app.versionsRows
+    appVersions: state.app.versionsRows,
```

Comment:
we can probably remove this now and do the translation to the rows and the sorting here in the component.